arch/arm/core/aarch32/Kconfig

```
@@ -40,6 +40,13 @@ config CPU_AARCH32_CORTEX_R
 	help
 	  This option signifies the use of a CPU of the Cortex-R family.
 
+config AARCH32_IMAGE_HEADER
+	bool "Add image header"
```

"Image header" can mean many different things.

Based on the description provided in the commit message, this should be named `ARM_ZIMAGE_HEADER` or similar.

```suggestion
config ARM_ZIMAGE_HEADER
	bool "zImage Header"
```

---

I have addressed this change.